### PR TITLE
Fix the color of the update dot on the settings icon

### LINKED ISSUE #929: Cog icon and update indicator looks like it is not rendering correctly
## Description

- The cog icon looks a bit jagged. Is it a rendering issue?
- Yellow highlight indicator should be bright at all times. In the current build it fades in and out as the cog icon is hovered in and out of. 

Screen Recording 2022-06-17 at 09 49 1

Screenshot 2022-05-23 at 10 43 1

## Acceptance criteria

- The cog icon should look crisp. If we cant fix, we can provide another icon.
- The yellow icon appears at 100% opcaity at all times while it is visible.

## Checklist
- [x] Include labels so that we can categorise your issue

### PR DESCRIPTION
**Pull request by tynandebold**
Signed-off-by: Tynan DeBold <thdebold@gmail.com>

## Description

Fixes #929 

## Development notes

I've removed the dot from the SVG and moved it to be a sibling element. That way, it isn't affected by the opacity of the icon and can be bright all the time.

## QA notes

Check the Gitpod link. The little dot should be bright. 

@GabrielComymQB regarding the icon not appearing crisp, I think this is a Retina display versus non-Retina issue. It looks really crisp on my laptop's screen (and in the screenshot it's not as crisp as what I see):

image

On my external 27" monitor, less so, though not terrible.

## Checklist

- [x] Read the [contributing](/CONTRIBUTING.md) guidelines
- [ ] Opened this PR as a 'Draft Pull Request' if it is work-in-progress
- [ ] Updated the documentation to reflect the code changes
- [ ] Added new entries to the `RELEASE.md` file
- [ ] Added tests to cover my changes

<a href="https://gitpod.io/#https://github.com/kedro-org/kedro-viz/pull/956"></a>

### CODE REVIEW

File: src/components/global-toolbar/global-toolbar.scss [hunk 1]
```diff
@@ -51,3 +51,13 @@
...
+
+.update-reminder-dot {
+  background-color: $color-accent-yellow;
+  border-radius: 50%;
+  height: 8px;
+  position: absolute;
+  right: 28px;
+  top: 31px;
+  width: 8px;
+}
```
Comment (cvetankanechevska) [inline src/components/global-toolbar/global-toolbar.scss:63]: 'dot' on the SettingsIcon is not positioned properly, here is the screenshot how it looks on my Mac display:

Screenshot 2022-07-05 at 13 58 32


I think `svg` and `span` elements need to be both positioned inside the same element, to be properly shown on different screen sizes.

**Review by cvetankanechevska — CHANGES_REQUESTED** [review]

**Comment by cvetankanechevska** [comment]
Here:

 -https://www.figma.com/file/K8WJMMTkvFHn6OZYg7Ypfz/Kedro-Design%3A-IN?node-id=4207%3A28411 
 -https://app.zeplin.io/project/5d5ea7a05efca76a74f7d0ea/screen/60be4d12b1f7ad11026a30a6

 we have a bit different SettingsIcon, should we change it now?

File: src/components/global-toolbar/global-toolbar.scss (hunk 1 above)
Comment (tynandebold) [reply src/components/global-toolbar/global-toolbar.scss:63]: Can you kill your frontend server and then restart it please? I have a feeling the code needs to compile for you to see it correctly.

**Comment by tynandebold** [comment]
> 

I'm not sure. Both of those look the same to me.

**Comment by cvetankanechevska** [comment]
> I'm not sure. Both of those look the same to me.

Sorry for my bad explanation, I was thinking from the one that we currently have, the edges are different from the icon in Zeplin and Figma.

File: src/components/global-toolbar/global-toolbar.scss (hunk 1 above)
Comment (cvetankanechevska) [reply src/components/global-toolbar/global-toolbar.scss:63]: Yes, it's fine now :D

**Review by cvetankanechevska — APPROVED** [review]
Looks great! 🎉

**Review by Huongg — APPROVED** [review]
Looks good to me too 👍

**Review by GabrielComymQB — APPROVED** [review]

**Pull request merged**